### ISSUE TITLE
Things to do before www launch

### ISSUE THREAD

**Issue by sammyjava** [legumeinfo/jekyll-legumeinfo#54]
Here is a list of things we decided need to be done before the www launch, edit this issue to add more.
- [X] host Tripal site on legacy.legumeinfo.org (in addition to www for the moment) [Nathan/Rex]
- [x] add genus/species resources or tools links from Jekyll site to legacy.legumeinfo.org where appropriate [?]
- [x] put gene family viewer into production on production URL with imjs query layer [Sven]
- [x] all-hands review of current dev site [Sam - issue posted]
- [x] add Google Analytics with legumeinfo.org code
- [x] update resources/tools to legacy or standalone URLs

Removed:
- implement standalone hosting for JBrowse, GBrowse - these will continue on legacy.legumeinfo.org. GCV standalone URL has been created and updated on resources and tools by SH.

**Comment by nathanweeks** [comment]
@adf had agreed to take the lead in completing the [legumeinfo/gcv@gcv.legumeinfo.org loader script](https://github.com/legumeinfo/gcv/blob/gcv.legumeinfo.org/microservices/redis_loader/docker-entrypoint-initdb.d/load.sh) so that it replicates the chado-dump-backed older GCV currently in production at legumeinfo.org/gcv2. Then any commits pushed to the gcv.legumeinfo.org branch will trigger a redeploy (at https://gcv.legumeinfo.org/) via a [GitHub Action workflow](https://github.com/legumeinfo/gcv/blob/gcv.legumeinfo.org/.github/workflows/deploy.yml).

**Comment by adf-ncgr** [comment]
yes, no progress yet but still weighing heavily on my conscience.

**Comment by nathanweeks** [comment]
@ctcncgr IIRC, you mentioned auto-generating a JBrowse 2 config.json for a subset of the LIS data store; if so, would it be feasible to expand & apply that approach to the entire data store to approximate the [gj]browse data sets and (subset of) tracks at legacy.legumeinfo.org?

**Comment by nathanweeks** [comment]
> implement standalone hosting for GCV, JBrowse, GBrowse (!) (to break dependence on legacy hosting) [Nathan?]

TBH, if the lights will be left on at `legacy` for a while yet, this could happen after the `www` launch (with relevant links at `www` pointing to applications at `legacy` in the interim)

**Comment by svengato** [comment]
> put gene family viewer into production on production URL with imjs query layer [Sven]

It really uses InterMineR instead of imjs.

**Comment by adf-ncgr** [comment]
@nathanweeks I just pushed an updated load.sh script but not entirely clear to me whether the redeploy has been triggered (seemingly not). Not sure if I need to have more specified in my environment to get this working. Perhaps you can school me on how this should work when convenient?

**Comment by adf-ncgr** [comment]
PS. I think I will break standalone GCV into a separate issue so we can carry on the discussion there

**Comment by sammyjava** [comment]
All checkboxes are checked. @nathanweeks is working on the go-live plumbing, @alancleary may be applying some non-blocking updates. The dev branch contains all my updates to finish off the list on this issue, so now closing it. We'll deal with the problems merging to main which seem to have appeared and are probably not a big deal.
